shared_model/validators/field_validator.hpp

```diff
@@ -183,7 +185,7 @@ namespace shared_model {
       // gap for future transactions
       time_t future_gap_;
       // time provider callback
-      TimeFunction time_provider_;
+      mutable TimeFunction time_provider_;
```

Comment:
Why not removing `mutable` here and `const` from `setTime(...)`?

Reply:
Because validaton methods are const, and they use setTime